src/content/docs/iast/overview.mdx

```diff
@@ -0,0 +1,114 @@
+---
+title: Introduction to New Relic IAST
+tags:
+    - New Relic IAST
+    - IAST
+    - Exploitable vulnerabilities
+metaDescription: Learn what New Relic IAST is and how you can use it.
+redirects:
+    - /docs/iast/use-iast
+---
+
+import iastSummary from 'images/iast_screenshot-crop_summary.webp'
+
+
+<Callout title="PREVIEW"> 
+    This feature is currently in preview. To celebrate it, we're offering you a 3-month free trial! The IAST free trial starts the moment you accept the pre-release software terms. 
```

Comment:
The wording in the second sentence feels odd to me and not inline with our doc site's usual voice and tone:

"To celebrate it, we're offering you a 3-month free trial!"

Can we tone this down a bit? :)

Reply:
I've just asked Krystle if we should maintain that preview note or not. I copied it from the [IAST page](https://docs.newrelic.com/docs/iast/use-iast/) we have now on the doc site. I'm going to wait for her answer before modifying it. Thanks!